apkid/rules/elf/packers.yara

```diff
@@ -372,6 +372,7 @@ rule promon_a : packer
   condition:
     is_elf and
     $rnd_libname and
+    not promon_b and
```

Comment:
Good thinking, though should this be `promon_a`?

Reply:
This was immediately spotted, thanks for double-checking. Check this out https://github.com/rednaga/APKiD/pull/235/commits/902e2f87f2b62244960296c61baaa3893c8c4af8

Reply:
Also, `not promon_a` condition won't match both Promon rules, just fyi

Reply:
Since you added this, can I assume that some `promon_b` samples were matching on `promon_a`? If this is the case, can any of the strings from `promon_a` be added here to bolster specificity? Again, this isn't needed if you have good insight on the elf section names, i.e. you know they're fairly unique to promon and not an artifact for some related build tool which promon happens to use.

Reply:
Added that for pure automatism. There's no need to add it.

Reply:
What is automatism? The rule is more clear if you don't include it assuming you're confident a sample should not hit on both rules.